202312-quarto-reports/2023-12-quarto-reports.qmd

```diff
@@ -0,0 +1,278 @@
+-   *Quarto has a greatly expanded range of document types*
+:::
+
+## Comparison  {.smaller}
+
+| Feature                                                       | Quarto | Rmd | Markdown |
+|:-----------------------------------------|:---------|:---------|:---------|
+| Markup syntax (`**` for emphasis, etc)                        | x      | x   | x        |
+| Render to multiple document types                             | x      | x   | x        |
+| Interweave text and code                                      | x      | x   | x        |
+| Interweave text and *executed* R code                         | x      | x   |          |
+| Execute *non-R* code                                          | x      |     |          |
+| Expanded set of document types (dashboards, manuscripts, ...) | x      |     |          |
+
+## How do I create a Quarto document?
+
+Let's start from this R script: <https://github.com/cct-datascience/workshop-wednesdays/blob/2023-quarto-reports/20231206-quarto-reports/mouse_report.R>
+
+## .R to .Qmd 
+
+::: incremental
+
+1. Create a Quarto document
+1. Move code chunks over
+1. Add descriptive text
+1. Add references
+1. Add figure/table cross-references and captions
+
+:::
+
+## How do I turn an .Rmd into a Quarto document?
+
+Example report document available here: <https://github.com/cct-datascience/workshop-wednesdays/blob/2023-quarto-reports/20231206-quarto-reports/mouse_report_RMD.Rmd>
+
+
+## Technical changes from .Rmd to Quarto
+
+::: incremental
+**Most .Rmd documents will render as-is!**
+
+-   Document-level options moved to header
+
+-   `output` changed to `format`
+
+-   Chunk-level options moved from ```` ```{r ....} ```` to `#|` within the chunk
+
+-   `knitr::convert_chunk_header(<file_path>)` helps with conversion
+
+-   <https://quarto.org/docs/faq/rmarkdown.html>
+:::
+
+
+## .Rmd to .Qmd
+
+::: incremental
+
+1. Run `knitr::convert_chunk_header`
+1. Add additional Quarto features 
+
+:::
+
+## Sharing Quarto reports
+
+| Hosted online with a shareable URL | Static documents to share person-to-person |
+|:-----------------------------------|:-----------------------------------|
+| Quarto Pub                         | Rendered HTML, pdf, docx outputs           |
+| Posit Connect                      | Source .qmd files                          |
+| GitHub                             |                                            |
+
+## QuartoPub
+
+::: incremental
+-   <https://quartopub.com/>
+
+-   Create a free account
+
+-   Published reports are **public**
+
+-   Example: <https://cct-datascience.quarto.pub/quarto-workshop-example/>
+:::
+
+## Posit Connect
+
+::: incremental
+-   <https://viz.datascience.arizona.edu>
+
+-   For **university research**; accounts available but restricted
+
+-   Published reports can be public or private
+
+-   Example: <https://viz.datascience.arizona.edu/quarto-workshop-example/>
+:::
+
+## GitHub
+
+::: incremental
+-   Render using the `gfm` output format
```

Comment:
There's also the option of GitHub pages with html output you _might_ mention.  I think running `quarto render mydoc.qmd` in terminal will give you this choice.  `gfm` won't work with all features of Quarto.

Reply:
I think I lean away from getting deep into GitHub pages rendering (I don't know if folks are likely to be set up for that), but good to mention!